src/Security/Authentication/Cookies/src/CookieAuthenticationHandler.cs

```diff
@@ -174,8 +182,6 @@ private async Task<AuthenticateResult> ReadCookieTicket()
                 return AuthenticateResult.Fail("Ticket expired");
             }
 
-            CheckForRefresh(ticket);
```

Comment:
I moved this to clarify that it's only relevant to Authenticate scenarios. SignIn and SignOut also call ReadCookieTicket, but only to populate the _sessionKey.